.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
```

Comment:
Yeah, I think in the meantime cross building built into sbt has appeared (independent on your actual Scala version)